vunit/builtins.py

```diff
@@ -44,13 +45,15 @@ def _add_files(self, pattern):
         """
         Add files with naming convention to indicate which standard is supported
         """
-        supports_context = self._simulator_class.supports_vhdl_2008_contexts() and self._vhdl_standard == "2008"
+        supports_context = self._simulator_class.supports_vhdl_contexts() and self._vhdl_standard in ["2008", "2019"]
```

Comment:
Instead of having `in ["2008", "2019"]` or `not in ["2008", "2019"]`, can we find some *cleaner* solution to *group* them?

- `self._vhdl_standard >= "2008"`. I honestly don't know whether this is feasible.
- `self._vhdl_standard_tag in ["2008p"]`. Alternatively, by adding a helper `check_vhdl_standard_tag(required_min_tag)` function: `check_vhdl_standard_tag("2008p")`.

Reply:
I did consider it for a while but postponed it for the future. Do you think we need to fix it before the release?

Reply:
No, I don't think we need to fix this before the release. I just tried to *visualize* how to implement such enhancement, and I wasn't sure about it.

Related to the comments about standard names last week, I think we should have all of the standard names, aliases and these *tags* defined in a single place. Then, when `from_argv` or `main` are executed, functions should check them ASAP, so that all other internal steps can rely on some fixed/known values. Functions that need to know the version, can check the *parsed version*, and functions that need to know the tag can check the *parsed tag*.

Reply:
I think we should have a vhdl_standard.py file defining a type for the vhdl standard with <= operators and similar instead of using strings. When adding support for VHDL 2019 it would have been a good opportunity to do it.